benmarks.py

```
@@ -0,0 +1,842 @@
+        'issueDate',
+        'currentAmountOutstanding',
+        'issueSubType',
+        'issueType',
+        'statusType',
+        'tier'
+    ]
+
+    for field in required_fields:
+        if treasury.get(field, None) is None:
+            message = 'Missing Required Field {}'.format(field)
+            upload_treasury_to_rejection_table(treasury, message)
```

Ruff (0.8.2)

Undefined name `upload_treasury_to_rejection_table`

(F821)